Tasks/AzureResourceManagerTemplateDeploymentV3/Tests/L0.ts

```diff
@@ -19,7 +19,7 @@ describe('Azure Resource Manager Template Deployment', function () {
     process.env['AGENT_HOMEDIRECTORY'] = process.env['AGENT_HOMEDIRECTORY'] || "C:\\temp\\agent\\home";
 	process.env['BUILD_SOURCESDIRECTORY'] = process.env['BUILD_SOURCESDIRECTORY'] || "C:\\temp\\agent\\home\\sources",
 	process.env['SYSTEM_DEFAULTWORKINGDIRECTORY'] = process.env['SYSTEM_DEFAULTWORKINGDIRECTORY'] || "C:\\temp\\agent\\home";
-	process.env["AGENT_TEMPDIRECTORY"] = process.env["AGENT_TEMPDIRECTORY"] || "C:\\temp\\agent\\home\\temp";
+	process.env["AGENT_TEMPDIRECTORY"] = process.env["AGENT_TEMPDIRECTORY"] || process.env["RUNNER_TEMP"] || process.env["TMP"] || "C:\\temp\\agent\\home\\temp";
```

Comment:
One of the issues I had was that the tasks would not run on my local machine. `process.env["RUNNER_TEMP"]` would allow running the tests in Github Action (which I highly recommend for visibility), previously the Azure DevOps pipelines where not open to the world. I possibly have access now that my account is invited in the MS tenant.

`process.env["TMP"]` allows running on local machine, since all machines have `TMP` defined but probably not `AGENT_TEMPDIRECTORY`